### ISSUE TITLE
Gather tokens for assignments in a separate directory called tokens

### ISSUE THREAD

**Issue by modelarious** [modelarious/FantasiaSource#39]
folder structure should be the following:
tokens for all successfully submitted assignments for an assignment key are stored in:
```/path/to/upload/${AssignmentKey}/tokens```

each directory (excluding the tokens directory from above) for an assignment key is going to be a UUID:
```/path/to/upload/${AssignmentKey}/${UUID}/```
inside each of those directories will be the original assignment, the tokenizer output and the generated reports (maybe could have a folder like the tokens one above for holding reports as long as I can identify them by which submission was theirs (could just put it the name of the report file))

**Comment by modelarious** [comment]
blocked by #13

**Comment by modelarious** [comment]
No longer collecting by assignment, so this is out of scope
